docs/gallery/gallery.rst

```diff
@@ -19,22 +19,22 @@ The following tutorials introduce core mathematical concepts
 provided by The Walrus, including the hafnian, loop hafnian,
 the permanent and the torontonian.
 
-.. customgalleryitem::
+.. gallery-item::
```

Comment:
Was the `customgalleyitem` specific to the TW docs before? I noticed that the `gallery-item`s look slightly different (larger, and the text is moved), but neither better nor worse.

Reply:
Yeah, looks like the shape and spacing is different.

Now: 
![image](https://user-images.githubusercontent.com/2959003/163405285-34b566fe-aef2-4552-afa5-f6c4876a47c7.png)

Before:
![image](https://user-images.githubusercontent.com/2959003/163405324-28804ec8-0fb7-4ed5-a0e6-8435e02ec084.png)

Reply:
Very minor change @Mandrenkov, but is it possible to make the tooltips optional when creating cards? I realized we removed this in several repos intentionally (The Walrus), but it remains in other repos (QML).

This isn't priority though, just nice to have.

Reply:
> Was the `customgalleyitem` specific to the TW docs before? I noticed that the `gallery-item`s look slightly different (larger, and the text is moved), but neither better nor worse.

Yeah, there was a slight divergence in the gallery items between PennyLane, Strawberry Fields, and The Walrus: I tried to unify all of them into a style which works for all repositories but I think I have over-fitted onto PennyLane. I'll see what I can do about the sizing to make everything fit on one row if that is desirable.

> Very minor change @Mandrenkov, but is it possible to make the tooltips optional when creating cards? I realized we removed this in several repos intentionally (The Walrus), but it remains in other repos (QML).

Sure, that should be easy to do. I figured the tool-tips were a feature (and not a 🐛) but we can certainly make them optional.

Reply:
> Yeah, there was a slight divergence in the gallery items between PennyLane, Strawberry Fields, and The Walrus: I tried to unify all of them into a style which works for all repositories but I think I have over-fitted onto PennyLane. I'll see what I can do about the sizing to make everything fit on one row if that is desirable.

Actually I wouldn't worry about this one too much for now! I think the most visible cards are on the SF and PL websites, so maybe we should wait until the theme is deployed everywhere before tweaking